src/src/pages/capabilities/capabilityTags/capabilityTagsSubForm.js

```diff
@@ -31,6 +46,48 @@ const CustomDropdown = function (props) {
   );
 };
 
+const CustomCheckbox = function (props) {
+  const { onChange } = props;
+  const [selectedOption, setSelectedOption] = useState(undefined);
+  let c1 = (Math.random() + 1).toString(36).substring(5);
+  let c2 = (Math.random() + 1).toString(36).substring(5);
```

Comment:
can you add a helper function called something like generateUniqueId in utils? Or simply just use uuid. At first glance I was like what the hell is this 😄

Reply:
removed this in its entirety.
We don't need the references, as we are not working with forms normally.